tsconfig.json

```diff
@@ -4,29 +4,29 @@
     "allowUnreachableCode": false,
     "downlevelIteration": true,
     "importHelpers": true,
-    "sourceMap": true,
+    "importsNotUsedAsValues": "error",
     "inlineSources": false,
+    "lib": ["dom", "ES2020"],
```

Comment:
Hey James - Sorry for not catching this earlier. Does this config mean that the polyfill won't support downlevel clients that don't have ES2020?

Reply:
No - this is just the tsconfig. This output is not published to NPM untouched - it passes through Babel first. We downlevel some of the artifacts published to NPM using Babel: https://github.com/js-temporal/temporal-polyfill/blob/d8dfacb47a562c127156bb53f2790e66d973181d/rollup.config.js#L147

Reply:
And this particular line was already present in tsconfig (see L19 on the "before" side).

Reply:
OK cool.